include/personality_test1.h

```diff
@@ -9,7 +9,7 @@ typedef struct PersonalityQuestion
 {
     /* 0x0 */ const u8 *question;
     /* 0x4 */ const MenuItem *answers;
-    /* 0x8 */ const u8 *effects;
+    /* 0x8 */ const u8 (*effects)[16];
```

Comment:
Maybe a good idea to bind this to `NUM_PERSONALITIES + 3` or modify the enum of personalities so we can bound this with a constant and allow hackers to increase/decrease these later.